CHANGELOG.md

```diff
@@ -4,6 +4,20 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.2.1
+
+This release contains bugfixes.
+
+**Notable commits**
+* [2f6b58a] Fix tlist on hypertable inserts inside CTEs
+* [7973b4a] Stop background worker on rename database
+* [32cc645] Fix loading the tsl license in parallel workers
+
+**Thanks**
+
+* @jamessewell for reporting and helping debug a segfault in last()
+* @piscopoc for reporting a segfault in time_bucket_gapfill
```

Comment:
Small small nitpick - do you mind noting which commits are related to these two issues?

Reply:
Not sure which they are, I think the second one is [e6c68f8] (@svenklemm ?)

Reply:
[034a0b0] is the bugfix for first/last